heroic-core/src/main/java/com/spotify/heroic/cluster/Update.kt

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2019 Spotify AB.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.heroic.cluster
+
+import eu.toolchain.async.Transform
+import java.net.URI
+import java.util.*
```

Comment:
I haven't read too much on it, but my understanding is its because Kotlin has more than just classes namespaced, so doing something like importing a bunch of functions from a package can be tedious. Java style guides generally do the wildcards for static imports for the same reason.